### ISSUE TITLE
Request/Future of this Repo ?

### ISSUE THREAD

**Issue by vabene1111** [tschwery/docker-ttrss#1]
Hey @tschwery 

if have used your repo to get my ttrss container up but i noticed i could not update to the latest ttrss versions since they changed their release setup (at least thats what i think).

I have updated the image in my fork of this repo https://github.com/vabene1111/docker-ttrss 

I would like to polish it up some more, but i am not sure if you would like to have them under this repo or if i should just continue working on my fork. You also dont state any license so i wanted to ask if you are fine with me creating the fork and maybe adding a proper free license if that is your wish.

It would be great if you could clarify how you wish this code should live on :)

**Comment by tschwery** [comment]
Hey,

Thanks for your issue. I am still using this repo for my ttrss instance and forgot to update them for the last 2 years.

I updated the URL to match what they are currently using (seems like ttrss migrated from Gitlab to Gogs since I deployed last time) and also updated PHP from 5.6 to 7. Concerning the license, I put a MIT license on there to clarify.

If you want to update or polish stuff, fork and go wild ! If you want to open PRs I will gladly read, consider and merge them.
